.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
 # Set default code owners for all files in repo
-* @mjhenkes @ryanthemanuel @tbiethman
+* @mjhenkes @ryanthemanuel @tbiethman @benbcai
```

```suggestion
* @ryanthemanuel @tbiethman @benbcai
```
might as well just make one pass